net-nds/389-ds-base/389-ds-base-1.4.2.15.ebuild

```diff
@@ -1,39 +1,40 @@
 # Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
+EAPI=7
 
-WANT_AUTOMAKE="1.13"
+WANT_AUTOMAKE="1.16"
 
-inherit user eutils multilib flag-o-matic autotools
-
-DESCRIPTION="389 Directory Server (core librares and daemons )"
-HOMEPAGE="http://www.port389.org/"
-SRC_URI="http://www.port389.org/sources/${P}.tar.bz2"
+inherit eutils multilib flag-o-matic autotools systemd db-use
 
+DESCRIPTION="389 Directory Server (core librares and daemons)"
+HOMEPAGE="https://directory.fedoraproject.org/"
+MY_P="${PN}-${PN}-${PV}"
+SRC_URI="https://pagure.io/389-ds-base/archive/${PN}-${PV}/${MY_P}.tar.gz"
+S="${WORKDIR}/${MY_P}"
 LICENSE="GPL-3+"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
```

Comment:
is this still working on x86? iirc upstream dropped support for that.

Reply:
What Operating systems are supported?

389 supports:

    Linux - Directory Server should build on:
        Fedora 4 and later ( x86 and x86_64 )
        Red Hat Enterprise Linux 3 and later ( x86 and x86_64 )
        others - debian, gentoo, ubuntu, more
    Solaris 2.8, 2.9 (32 bit and 64 bit) ( sparc )
    HP/UX 11 ( pa-risc and ia64 )
According to: https://directory.fedoraproject.org/docs/389ds/FAQ/faq.html

Caveat - I haven't tried it!

Reply:
yeah same caveat for me. i was basing my statement around the fact that fedora dropped i686 support after 1.3.
i have looked around a bit and found this (very long) bug report: https://bugzilla.redhat.com/show_bug.cgi?id=1544386 which in turn links to https://bugzilla.redhat.com/show_bug.cgi?id=1530832 which describes the core issue in comment 1 without having to scroll through everything

Reply:
> 
> 
> yeah same caveat for me. i was basing my statement around the fact that fedora dropped i686 support after 1.3.
> i have looked around a bit and found this (very long) bug report: [bugzilla.redhat.com/show_bug.cgi?id=1544386](https://bugzilla.redhat.com/show_bug.cgi?id=1544386) which in turn links to [bugzilla.redhat.com/show_bug.cgi?id=1530832](https://bugzilla.redhat.com/show_bug.cgi?id=1530832) which describes the core issue in comment 1 without having to scroll through everything

I see, excellent thank you.

I will remove x86 support then.

I was kind of rushing to get this PR ready before the packet was removed, so didn't quite do as much digging as I would normally!